packaging/linux/post_install.sh

```diff
@@ -7,12 +7,44 @@
 
 set -u
 
-# Create the /keybase root dir, if it doesn't already exist. We can't use a
-# regular [ -e ... ] check, because stat'ing /keybase fails with a permissions
-# error when kbfsfuse is mounted, so that check always returns false. Instead
-# we check whether mkdir succeeds.
-if mkdir /keybase &>/dev/null ; then
-  chmod 777 /keybase
+# Create the /keybase symlink to the first mountpoint, if it doesn't
+# already exist. We can't use a regular [ -e ... ] check, because
+# stat'ing /keybase fails with a permissions error when kbfsfuse is
+# mounted, so that check always returns false. Instead we check
+# whether mkdir succeeds.
+rootlink="/keybase"
+vardir="/var/lib/keybase"
+mount1="$vardir/mount1"
+sample="/opt/keybase/mount-readme"
+khuser="keybasehelper"
+khbin="/usr/bin/keybase-mount-helper"
+
+# Create the keybasehelper system user, without login privileges.
+if useradd --system -s /bin/false -U -M $khuser &> /dev/null ; then
+    echo Created $khuser system user for managing mountpoints.
+fi
+
+chown $khuser:$khuser $khbin
+chmod 4755 $khbin
+if mkdir $vardir &> /dev/null ; then
+    ln -s $sample $mount1
+    chown -R $khuser:$khuser $vardir
+fi
+
+currlink=`readlink $rootlink`
+if [ -z "$currlink" ] ; then
+    if fusermount -uz /keybase &> /dev/null ; then
+        # Remove any existing legacy mount.
+        echo Unmounting /keybase...
+        if killall kbfsfuse &> /dev/null ; then
```

Comment:
The `killall` here probably isn't a big deal, since we're trying to only run it the one time you install this symlink. But just in case: Is there any way to make the process owning that FUSE mount exit gracefully (or maybe just leave it running?), to avoid any corner cases where we accidentally kill some unrelated processes we shouldn't have?